esp-hal-common/src/i2s.rs

```diff
@@ -475,11 +476,13 @@ where
     ) -> Self
     where
         IP: I2sPeripheral,
+        I: Instance<T>,
```

Comment:
I think this where bound can be removed because its already bound by the `impl` block.

Reply:
Thanks, you are right!